runtime-sdk/src/modules/rofl/mod.rs

```diff
@@ -204,6 +199,13 @@ impl<Cfg: Config> Module<Cfg> {
             }
         }
 
+        if body.policy.endorsements.len() > Cfg::MAX_ENDORSEMENT_POLICY_ATOMS {
```

Comment:
Isn't this included in the next check?

Reply:
Yes, I guess it is just saving some recursion. Now that I think of it, depth could (should?) also be checked here.

Reply:
Agree, first check depth, then atoms. Could also improve in such a way that this check stops once depth or counter is exceeded (this way you always count at most 32 atoms).